assets/app/js/controllers/conclusionDialog.js

```diff
@@ -204,7 +204,12 @@ class ConclusionDialog extends Controller {
           data: $.unique(license_expressions),
           multiple: true,
           placeholder: 'Enter Expression',
-          tags: true
+          tags: true,
+          language: {
+            'noResults': function () {
+              return (`SCWB did not find any license expressions in the scan that applies to ${conclusion.path}`);
```

Comment:
In some cases, this `conclusion.path` value is very long and is awkward to show in a small menu item. 

Instead of `${conclusion.path}`, we should have "the current path".